client/components/jetpack/intro-pricing-banner/style.scss

```diff
@@ -8,33 +11,48 @@
 	justify-content: center;
 
 	.intro-pricing-banner__discount,
-	.intro-pricing-banner__guarantee {
+	.intro-pricing-banner__guarantee,
+	.intro-pricing-banner__agencies {
 		display: flex;
 		flex-direction: row;
 		align-items: center;
 
 		font-size: $font-body;
+
+		a {
+			color: var( --color-primary-100 );
+			text-decoration: underline;
+			text-underline-offset: 3px;
```

Comment:
FYI this is supported for all major browsers, with the sole exception of Firefox for Android: https://caniuse.com/?search=text-underline-offset

It is also not extremely important that this works

Reply:
Does it make sense to leave some short inline comment on the side explaining that?

Reply:
I could see that being helpful, I'll do that 😄